chain-impl-mockchain/src/evm/mod.rs

```diff
@@ -216,6 +244,48 @@ impl Readable for EvmTransaction {
                     access_list,
                 })
             }
+            #[cfg(feature = "evm")]
+            1 => {
+                // CREATE2 Transaction
+                let caller = read_address(buf)?;
+                let value = read_u256(buf)?;
+                let init_code = read_bytecode(buf)?;
+                let salt = read_h256(buf)?;
+                let gas_limit = read_gas_limit(buf)?;
+                let access_list = read_access_list(buf)?;
+
+                buf.expect_end()?;
+
+                Ok(EvmTransaction::Create2 {
+                    caller,
+                    value,
+                    init_code,
+                    salt,
+                    gas_limit,
+                    access_list,
+                })
+            }
+            #[cfg(feature = "evm")]
+            2 => {
+                // CALL Transaction
+                let caller = read_address(buf)?;
+                let address = read_address(buf)?;
+                let value = read_u256(buf)?;
+                let data = read_bytecode(buf)?;
+                let gas_limit = read_gas_limit(buf)?;
+                let access_list = read_access_list(buf)?;
+
+                buf.expect_end()?;
```

Comment:
same as above (there is also one in `Create` that I cannot flag as it's already committed to the repo)